README.md

```diff
@@ -1,2 +1,171 @@
+
+
+### Installing Julia
+To obtain Julia, go to https://julialang.org/downloads/ and download the latest
+stable release (v1.7.3 as of 2022-06-28; neither use the LTS release nor
+Julia Pro). Then, follow the
+[platform-specific instructions](https://julialang.org/downloads/platform/)
+to install Julia on your machine. Note that there is no need to compile anything
+if you are using Linux, MacOS, or Windows.
+
+After the installation, open a terminal and start the Julia *REPL*
+(i.e., the interactive prompt) with
+```shell
+julia
+```
+
+### Installing the required Julia packages
+To run the scripts in the [examples](examples/) directory and allow for
+fully reproducible results, we have used Julia's package manager
+to pin all packages to a fixed release. This ensures that you always have a
+Julia environment in which all the results presented were created.
+
+If you have not done it yet, clone the repository where this code is stored:
+```shell
+git clone https://github.com/trixi-framework/talk-2022-juliacon_toolchain.git
+```
+Then, navigate to your repository folder and install the required packages:
+```shell
+cd talk-2022-juliacon_toolchain
+julia --project=. -e 'using Pkg; Pkg.instantiate()'
+```
+This will download and build all required packages, including the ODE package
+[OrdinaryDiffEq](https://github.com/SciML/OrdinaryDiffEq.jl), the visualization
+package [GLMakie](https://github.com/JuliaPlots/Makie.jl/tree/master/GLMakie),
+the mesh generator [HOHQMesh.jl](https://github.com/trixi-framework/HOHQMesh.jl),
+and [Trixi](https://github.com/trixi-framework/Trixi.jl).
+The `--project=.` argument tells Julia to use the `Project.toml`
+and `Manifest.toml` files from this repository to figure out which packages to install.
+
+Once the initialization and installation is complete you must start Julia with the
+`--project` flag set to your local clone of this repository
+```shell
+julia --project=@.
+```
+
+
+## Mesh created with tools from HOHQMesh.jl
+To reproduce the figures and create the mesh file output execute from the REPL
+```julia
+include(joinpath("examples", "interactive_cylinder_with_sine_walls.jl"))
+```
+This will create the directory `out` where the mesh file is saved.
+
+## Simulation with Trixi.jl
+The elixir file described in the presentation to setup and run a simulation
+of Mach 2 flow over a cylinder is `elixir_euler_mach2_cylinder.jl`.
+To run the simulation up to a final time of 0.5, execute from the REPL
+```julia
+using Trixi
+trixi_include(joinpath("examples", "elixir_euler_mach2_cylinder.jl"), tspan=(0.0,0.5))
+```
+where the final time is adjusted within the `trixi_include` call.
+This simulation to the final time 0.5 takes approximately 20 minutes on a single thread.
+To visualize the solution `sol` at the final time execute
+```julia
+using GLMakie
+pd = PlotData2D(sol)
+plot(pd["rho"])
+```
+
+## Combined script
+The script `build_mesh_and_run_mach2_cylinder.jl` executes the entire toolchain
```

Comment:
These scripts seem to contain a lot of repeated code. Is it possible to just write the "build mesh" and "run Trixi" scripts and then call them each sequentially for the "combined" script?

Reply:
That's a good point. I could just `include` the two scripts and then the `Trixi2Vtk` and `plot` calls are the only new things in the complete script.

Reply:
Awesome! Glad to hear.